docs/operations/single-server.md

```diff
@@ -44,6 +47,21 @@ The example configurations run the Druid Coordinator and Overlord together in a
 
 While example configurations are provided for very large single machines, at higher scales we recommend running Druid in a [clustered deployment](../tutorials/cluster.md), for fault-tolerance and reduced resource contention.
 
+## Druid auto start
+
+Setting up a new Druid cluster can sometimes be complicated as there are several runtime properties required by each service. Each process must also be given the appropriate jvm arguments so that the system can perform optimally.
+
+`start-druid` is a generic launch script capable of starting any set of Druid services on a server.
+It accepts optional arguments such as list of services, total memory and a config directory to override default jvm arguments and service-specific runtime properties.
+All other reference configurations (e.g. `micro`, `small`, `xlarge`) can be obtained by passing the
```

Comment:
I would cut this sentence, I think it does not add much other than some confusion.

Reply:
Removed.